tcms/testplans/static/testplans/js/get.js

```diff
@@ -30,6 +30,7 @@ $(document).ready(function() {
         }
         drawTestCases(allTestCases.confirmed, testPlanId, permissions);
         treeViewBind();
+        toolbarEvents();
```

Comment:
I don't understand why the binding needs to be inside the API call filtering test cases ? Why not at an upper level ? From what I can see this checkbox just iterates over elements and sets them checked or unchecked. If there's nothing loaded then nothing will happen, right ?

Reply:
I missed that it is in the API call response. I just aimed for document.ready function and saw bind method. You are right, I will move it out.